pontoon/checks/libraries/pontoon.py

```diff
@@ -65,7 +65,7 @@ def run_checks(entity, string):
                 )
 
     # Prevent empty translation submissions if not supported
-    if resource_ext not in {'properties', 'ini', 'dtd'} and string == '':
+    if resource_ext not in {'properties', 'ini', 'dtd', 'inc'} and string == '':
```

Comment:
[nit] Maybe it's the time to move to create a constant value e.g. EMPTY_TRANSLATION_FORMATS and move them to `pontoon.checks.__init__`?